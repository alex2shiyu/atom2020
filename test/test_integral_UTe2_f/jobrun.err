/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8): No such file or directory
/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8)
which: no manpath in (/share/home/pengsy/soft/intel-install/compilers_and_libraries_2019.5.281/linux/mpi/intel64/bin/:/share/home/pengsy/soft/intel-install/bin:/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/sbin:/opt/ibutils/bin)
/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8): No such file or directory
/share/home/pengsy/.bashrc: line 3: warning: setlocale: LC_CTYPE: cannot change locale (en_US.UTF-8)
Python 3.6.9 :: Anaconda, Inc.
Traceback (most recent call last):
  File "/share/home/pengsy/work/atom2020/src/main.py", line 113, in <module>
    pro += ir.dim/ir.nrank * np.conjugate(ir.matrices[irank][0,0])*manybody_umat[irank]
ValueError: operands could not be broadcast together with shapes (10,10) (91,91) (10,10) 
