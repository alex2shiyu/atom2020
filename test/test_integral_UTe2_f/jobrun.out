Job starts at Fri May 29 13:44:59 CST 2020
Job ends at Fri May 29 18:27:44 CST 2020
